test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -122,8 +122,9 @@ def test_prologue_internal_error_offline_vnodes(self):
         """
         attr = {'resources_available.mem': '2gb',
                 'resources_available.ncpus': '1'}
-        self.server.create_vnodes(self.hostC, attr, 3, self.momC, delall=True,
-                                  usenatvnode=True)
+        if not self.momC.is_cpuset_mom():
+            self.server.create_vnodes(self.hostC, attr, 3, self.momC,
+                                      delall=True, usenatvnode=True)
```

Comment:
We probably should skip the test if the mom is a cpuset mom.  With what you have here, the test will expect 3 vnodes, and will have the number of vnodes the cpuset om has natively.